baybe/utils/dataframe.py

```diff
@@ -337,10 +338,8 @@ def fuzzy_row_match(
             dataframe.
         parameters: List of baybe parameter objects that are needed to identify
             potential tolerances.
-        numerical_measurements_must_be_within_tolerance: If ``True``, numerical
-            parameters are matched with the search space elements only if there is a
-            match within the parameter tolerance. If ``False``, the closest match is
-            considered, irrespective of the distance.
+        on_tolerance_violation: The mode determining what how to handle a missing
```

Comment:
Either add some words about the different modes here or refer to the place where you described them.